### PR TITLE
Feature/remove old datastore implementation

### PR DESCRIPTION
**Pull request by ElRaptorus**
## What did you change?

- Remove any and all references, configs and dependencies to obsolete packages, such as `core_contracts` or `foundation`
- Move the process file import from an ioc-based logic, to a function located in the test-fixture-provider
- Add a config for the new timer-persistence layer
- Cleanup setup scripts

## How can others test the changes?

Run the test and be amazed just how incredibly fast they are, compared to before 🤘 

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

**Comment by ElRaptorus** [comment]
https://github.com/process-engine-pro/organization/issues/174
https://github.com/process-engine-pro/organization/issues/177
https://github.com/process-engine-pro/organization/issues/181

File: _integration_tests/src/test_fixture_provider.ts [hunk 1]
```diff
@@ -131,4 +109,54 @@ export class TestFixtureProvider {
       identity: username,
     };
   }
+
+  private async importProcessFiles(): Promise<void> {
```
Comment (5ebastianMeier) [inline _integration_tests/src/test_fixture_provider.ts]: * Unterstrich an privaten Methoden

File: _integration_tests/src/test_fixture_provider.ts [hunk 2]
```diff
@@ -131,4 +109,54 @@ export class TestFixtureProvider {
...
+
+    const dummyIdentity: IIdentity = await identityService.getIdentity('dummyToken');
+    const dummyContext: ExecutionContext = new ExecutionContext(dummyIdentity);
+
+    for (const processFileName of processFileNames) {
+      await this.registerProcess(dummyContext, processFileName, importService);
+    }
+  }
+
+  private async registerProcess(dummyContext: ExecutionContext, processFileName: string, importService: IImportProcessService): Promise<void> {
```
Comment (5ebastianMeier) [inline _integration_tests/src/test_fixture_provider.ts]: * Unterstrich an privaten Methoden

File: _integration_tests/src/test_fixture_provider.ts [hunk 3]
```diff
@@ -131,4 +109,54 @@ export class TestFixtureProvider {
...
+    await importService.importBpmnFromFile(dummyContext, processFilePath, true);
+  }
+
+  /**
+   * Generate an absoulte path, which points to the bpmn directory.
+   *
+   * Checks if the cwd is "_integration_tests". If not, that directory name is appended.
+   * This is necessary, because Jenkins uses a different cwd than the local machines do.
+   */
+  private getBpmnDirectoryPath(): string {
```
Comment (5ebastianMeier) [inline _integration_tests/src/test_fixture_provider.ts]: * Unterstrich an privaten Methoden

**Pull request merged**